doc/source/user_guide/numpy_images.rst

```diff
@@ -168,43 +167,43 @@ These conventions are summarized below:
   =========================   ========================================
 
 
-Many functions in scikit-image operate on 3D images directly:
+Many functions in ``scikit-image`` can operate on 3D images directly:
 
     >>> im3d = np.random.rand(100, 1000, 1000)
     >>> from skimage import morphology
     >>> from scipy import ndimage as ndi
     >>> seeds = ndi.label(im3d < 0.1)[0]
     >>> ws = morphology.watershed(im3d, seeds)
 
-In many cases,
-the third imaging dimension has lower resolution than the other two.
-Some scikit-image functions provide a ``spacing`` keyword argument
-to process these images:
+In many cases, however, the third spatial dimension has lower resolution
+than the other two. Some ``scikit-image`` functions provide a ``spacing``
+keyword argument to help handling this situation:
 
     >>> from skimage import segmentation
     >>> slics = segmentation.slic(im3d, spacing=[5, 1, 1], multichannel=False)
 
-
-Other times, processing must be done plane-wise. When planes are the
-leading dimension, we can use the following syntax:
+Other functions may be lacking support for any non-2D images, and have to be used
+for processing of such data in plane-wise manner. When planes are stacked
```

Comment:
I prefer the previous phrasing here, "other times...", because sometimes the function supports 3D, but not anisotropy. For example, I often do plane-wise watershed.

The added sentence ("when planes are stacked...", is a good change.